### DISCUSSION TITLE
Zooz 800 random controller crash

### DISCUSSION THREAD

**Discussion by duylong** [zwave-js/zwave-js#7003] [Request Support / Investigate Issue] [answered]
### Checklist

- [X] I have read and followed the above instructions
- [X] I have checked the [troubleshooting section](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index) and my problem is **not** described there.

- [X] I have read the [changelog](https://github.com/zwave-js/node-zwave-js/blob/master/CHANGELOG.md) and my problem was **not** mentioned there or the fix **did not work**.

### Describe the issue

I tried to change my zooz 700 controller (ZST10) to a zooz 800 (ZST39 LR).

For some reason, my new controller often crashes and I have to unplug and re-plug the key each time. The "soft reset" and "Serial port reopened" do not seem to work.

I upgraded the firmware to version 1.30 (based on SDK 7.21.3), but it is no better.

Is this problem known on this firmware? In the meantime, I went back to my zooz 700 which does not have this problem.

### Software versions

zwave-js-ui: 9.14.5.a2817c3
zwave-js: 12.12.3
Using with Docker

### Checklist

- [X] I [made sure](https://zwave-js.github.io/node-zwave-js/#/troubleshooting/index?id=rule-1-use-the-driver-log-luke) to provide a **driver log** on level **debug**.

- [ ] The log includes a re-interview of the problematic device _(if applicable)_.

- [ ] The log includes the problematic interaction with the device _(if applicable)_.

- [ ] I provided the node ID of the problematic device _(if applicable)_.

### Upload Logfile

[zwavejs_current.zip](https://github.com/user-attachments/files/16175539/zwavejs_current.zip)

**Comment by AlCalzone** [comment, marked as answer]
See https://github.com/zwave-js/node-zwave-js/discussions/6874
